iOS/UI/Reader/Page/ReaderPageView.swift

```diff
@@ -167,6 +172,13 @@ class ReaderPageView: UIView {
                         image = processedImage
                     }
                 }
+                if UserDefaults.standard.bool(forKey: "Reader.cropBorders") {
+                    let processor = CropBordersProcessor()
+                    let processedImage = processor.process(image)
+                    if let processedImage = processedImage {
```

Comment:
for future reference (I'll add this to the swiftlint rules in the future as I need to change it throughout the app)
```suggestion
                    if let processedImage {
```